ape_vyper/_cli.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import ape
+import click
+from ape.cli import ape_cli_context
+
+
+@click.group
+def cli():
+    """`vyper` command group"""
+
+
+@cli.command(short_help="Flatten select contract source files")
+@ape_cli_context()
+@click.argument("CONTRACT", type=click.Path(exists=True, resolve_path=True))
+@click.argument("OUTFILE", type=click.Path(exists=False, resolve_path=True, writable=True))
+def flatten(cli_ctx, contract: Path, outfile: Path):
+    """
+    Flatten a contract into a single file.
+    """
+    with Path(outfile).open("w") as fout:
+        content = ape.compilers.flatten_contract(
```

Comment:
Might want to do something like 


```suggestion
        content = ape.compilers.vyper.flatten_contract(
```

Because it's kinda weird this would work for solidity files

Reply:
Yeah, it's already implemented in ape-solidity.  That's kind of why I wanted to do a generic CLI command rather than a `vyper` command.

Reply:
I do like the idea of the command in core, my suggestion would be:

```
ape pm flatten <rest>
```